packages/node-cli/src/commands/account/savetoken/AccountSaveTokenAction.js

```diff
@@ -73,11 +72,8 @@ module.exports = class AccountSaveTokenAction extends BaseAction {
 		if (validationErrors.length > 0) {
 			throw new CLIException(ValidationErrorsFormatter.formatErrors(validationErrors));
 		}
-		// If url is system.netsuite.com, we must not pass it to the sdk. If it's anything else, we must add developmentMode flag
 		if (params[COMMAND.OPTIONS.URL] === GENERIC_NETSUITE_DOMAIN) {
 			delete params[COMMAND.OPTIONS.URL];
```

Comment:
Is it ok to pass an account specific url, will it work the same way?

Reply:
Yes, because the params dev was only used in AuthenticationUtils  for adding the dev flag to the Java CLI command.
Since we have deleted the need of this flag we could delete it also here.

Reply:
Sorry I think I haven't been clear in my question, which I'll try to make it again. Will the account:savetoken work properly providing the tokenid/tokensecret and a production account specific url?
I guess yes, but just I wanted your confirmation.